src/routes/routeConfig.ts

```diff
@@ -53,6 +59,28 @@ export const PUBLIC_ROUTE_CONFIG: Record<string, IRoute> = {
     title: 'Register - BeeQuant',
     component: Register,
   },
+
+  [ROUTE_KEY.RESET_PASSWORD_INITIATION]: {
+    path: '/reset-password-initiation',
+    name: 'Reset Password',
+    title: 'Reset Password - BeeQuant',
+    component: ResetPasswordInitiationPage,
+  },
+
+  [ROUTE_KEY.RESET_PASSWORD_FORM]: {
+    path: '/reset-password-form',
+    name: 'Reset Password Form',
+    title: 'Enter New Password - BeeQuant',
+    component: ResetPasswordForm,
+  },
+
+  [ROUTE_KEY.PASSWORD_RESET_SUCCESS]: {
+    path: '/password-reset-success',
```

Comment:
since our url design is more Restful style, it's better to be like `/password/reset/success`